paramgen_dest.go

```diff
@@ -9,24 +9,6 @@ import (
 
 func (DestinationConfig) Parameters() map[string]sdk.Parameter {
 	return map[string]sdk.Parameter{
-		"caCert": {
-			Default:     "",
-			Description: "caCert is the path to the CA certificate to use for TLS",
-			Type:        sdk.ParameterTypeString,
-			Validations: []sdk.Validation{},
-		},
-		"clientCert": {
-			Default:     "",
-			Description: "clientCert is the path to the client certificate to use for TLS",
-			Type:        sdk.ParameterTypeString,
-			Validations: []sdk.Validation{},
-		},
-		"clientKey": {
-			Default:     "",
-			Description: "clientKey is the path to the client key to use for TLS",
-			Type:        sdk.ParameterTypeString,
-			Validations: []sdk.Validation{},
-		},
 		"delivery.appID": {
```

Comment:
`delivery.contentType` is also missing in the readme (well, it's named `contentType`).